server/broadcast.go

```diff
@@ -872,8 +872,8 @@ func getSegDurMsString(seg *stream.HLSSegment) string {
 
 func isNonRetryableError(e error) bool {
 	foundErr := false
-	for _, v := range ffmpeg.LPMSErrors {
-		if e.Error() == v.Desc {
+	for _, v := range ffmpeg.NonRetryableErrs {
```

Comment:
Iterating through the slice is probably ok for now since it is still small, but we'll want to keep an eye on this if the slice gets larger since isNonRetryableError() is called for every single segment.

Reply:
Did a quick fix for it https://github.com/livepeer/go-livepeer/pull/1823/commits/e30a8087246f88c8f6081ab1b54365a036fa701d and tested that it works.

@yondonfu Still good to merge this PR? (I'm cool with keeping it till https://github.com/livepeer/go-livepeer/pull/1823/commits/1b6a7e3b617459cc43cdefb0fbd82949b82dcc91 and doing the bool map change in LPMS sometime later)

Reply:
> doing the bool map change in LPMS sometime later

👍

Reply:
dang it, saw your comment rn (github comment refresh is weird)
but will move the map code into LPMS when I get a chance